src/main/java/com/cinema/Main.java

```diff
@@ -62,17 +63,30 @@ public static void main(String[] args) throws AuthenticationException {
                 LocalDate.now().plusDays(1)).forEach(System.out::println);
 
         UserService userService = (UserService) injector.getInstance(UserService.class);
-        User alcapone = new User();
-        alcapone.setEmail("alcaponchik@porishau.com");
-        alcapone.setPassword("777");
-        userService.add(alcapone);
+        User panas = new User();
+        panas.setEmail("panas@u.com");
+        panas.setPassword("777");
+        userService.add(panas);
         AuthenticationService authenticationService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
-        User mclane = new User();
-        mclane.setEmail("oreshek@cop.com");
-        mclane.setPassword("911");
-        authenticationService.register(mclane.getEmail(), mclane.getPassword());
-        authenticationService.login(mclane.getEmail(), mclane.getPassword());
-        System.out.println(userService.findByEmail(mclane.getEmail()).get());
+        User onic = new User();
+        onic.setEmail("onic@u.com");
+        onic.setPassword("911");
+
+        authenticationService.register(onic.getEmail(), onic.getPassword());
+
+        authenticationService.login(onic.getEmail(), onic.getPassword());
+        System.out.println(userService.findByEmail(onic.getEmail()).get());
+        User foundUser = userService.findByEmail("onic@u.com").get();
+        System.out.println("My found user: " + foundUser);
+
+        ShoppingCartService shoppingCartService =
+                (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+        shoppingCartService.registerNewShoppingCart(panas);
```

Comment:
let's move this login to authenticationService.register()

Reply:
Done